pkgs/misc/vscode-extensions/vscode-utils.nix

```
@@ -28,7 +28,8 @@ let
 
     installPrefix = "share/vscode/extensions/${vscodeExtUniqueId}";
 
-    buildInputs = [ unzip ] ++ buildInputs;
+    nativeBuildInputs = [ unzip ];
```

Probably needs

```suggestion
    nativeBuildInputs = (a.nativeBuildInputs or []) ++ [ unzip ];
```

or use a default argument in the parameter list.